pkg/provider/azure_loadbalancer_test.go

```diff
@@ -7139,33 +7141,33 @@ func TestGetEligibleLoadBalancers(t *testing.T) {
 		{
 			description: "should respect service annotation",
 			svc:         getTestService("test", v1.ProtocolTCP, map[string]string{consts.ServiceAnnotationLoadBalancerConfigurations: "A ,b"}, false),
-			lbConfigs: []MultipleStandardLoadBalancerConfiguration{
+			lbConfigs: []ratelimitconfig.MultipleStandardLoadBalancerConfiguration{
```

Comment:
Why do we use ratelimitconfig. and azureconfig. at the same time?

Reply:
I will rename it to azureconfig